api4/cloud.go

```diff
@@ -141,6 +143,33 @@ func getCloudProducts(c *Context, w http.ResponseWriter, r *http.Request) {
 	w.Write(json)
 }
 
+func getCloudLimits(c *Context, w http.ResponseWriter, r *http.Request) {
+	if c.App.Channels().License() == nil || !*c.App.Channels().License().Features.Cloud {
+		c.Err = model.NewAppError("Api4.getCloudLimits", "api.cloud.license_error", nil, "", http.StatusNotImplemented)
+		return
+	}
+
+	if !c.App.Config().FeatureFlags.CloudFree {
```

Comment:
Does this feature flag indicate the user is on a cloud-free instance? If so, do we have the ability to change feature flags at runtime?

In Boards should we check for `License().Features.Cloud` and `FeatureFlags.CloudFree` before calling the `/limits` endpoint to avoid erros?